docs/building-deployer-helm.md

```diff
@@ -107,13 +113,20 @@ metadata:
 spec:
   descriptor:
     type: Wordpress
-    version: '6'
+    version: '6.0.0'
```

Comment:
perhaps also use `9.0.3` version for consistency?

Reply:
Done!